src/main/java/com/leets/xcellentbe/domain/comment/domain/Comment.java

```diff
@@ -0,0 +1,48 @@
+package com.leets.xcellentbe.domain.comment.domain;
+
+import java.util.UUID;
+
+import com.leets.xcellentbe.domain.post.domain.Post;
+import com.leets.xcellentbe.domain.shared.BaseTimeEntity;
+import com.leets.xcellentbe.domain.shared.DeletedStatus;
+import com.leets.xcellentbe.domain.user.domain.User;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.validation.constraints.NotNull;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Comment extends BaseTimeEntity {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.UUID)
+	private UUID CommentId;
+
+	@NotNull
+	@ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "writer_id")
+	private User writer;
```

Comment:
```suggestion
	@NotNull
	@ManyToOne(fetch = FetchType.LAZY)
        @JoinColumn(name = "writer_id")
	private User writer;
```

특별히 들여쓰기 한 이유가 없으시면 줄 맞춰주시면 좋을 것 같습니다! 😀

Reply:
IDE 상으로는 들여 쓰기가 동일하게 되어있어서 놓쳤네요 감사합니다!